modules/olmv1-creating-a-service-account.adoc

```diff
@@ -0,0 +1,138 @@
+// Module included in the following assemblies:
+//
+// * extensions/ce/managing-ce.adoc
+
+:_mod-docs-content-type: PROCEDURE
+
+[id="olmv1-creating-a-service-account_{context}"]
+= Creating a service account to manage cluster extensions
+
+Unlike {olmv0-first}, {olmv1} does not have permissions to install, update, and manage cluster extensions. Cluster administrators must create a service account and assign the role-based access controls (RBAC) required to install, update, and manage cluster extensions.
+
+[IMPORTANT]
+====
+include::snippets/olmv1-known-issue-service-accounts.adoc[]
+====
+
+.Prerequisites
+
+* Access to an {product-title} cluster using an account with `cluster-admin` permissions.
+
+.Procedure
+
+. Create a service account, similar to the following example:
++
+[source,yaml]
+----
+apiVersion: v1
+kind: ServiceAccount
+metadata:
+  name: <extension>-installer
+  namespace: <namespace>
+----
++
+.Example `extension-service-account.yaml` file
+[%collapsible]
+====
+[source,yaml]
+----
+apiVersion: v1
+kind: ServiceAccount
+metadata:
+  name: pipelines-installer
+  namespace: pipelines
+----
+====
+
+. Apply the service account by running the following command:
++
+[source,terminal]
+----
+$ oc apply -f extension-service-account.yaml
+----
+. Create a cluster role and assign RBAC, similar to the following example:
++
+[WARNING]
+====
+The following cluster role does not follow the principle of least privilege. This cluster role is intended for testing purposes only. Do not use it on production clusters.
+====
++
+[source,yaml]
+----
+apiVersion: rbac.authorization.k8s.io/v1
+kind: ClusterRole
+metadata:
+  name: <extension>-installer-clusterrole
+rules:
+- apiGroups: ["*"]
+  resources: ["*"]
+  verbs: ["*"]
+----
++
+.Example `pipelines-cluster-role.yaml` file
+[%collapsible]
+====
+[source,yaml]
+----
+apiVersion: rbac.authorization.k8s.io/v1
+kind: ClusterRole
+metadata:
+  name: pipelines-installer
```

Comment:
@michaelryanpeter per your change, it should be `pipelines-installer-clusterrole`

Reply:
@kuiwang02 addressed here: https://github.com/openshift/openshift-docs/pull/82245/commits/f8207a933bed47c88c904a0e822ab241a8837b51

Thank you!

Reply:
New commit link to the updated content: https://github.com/openshift/openshift-docs/pull/82245/commits/b9eb18737ceb3b9f1deec8e59d0fb603641aaba7